### PR TITLE
[Internal] Change log: Adds recommended version

### PR DESCRIPTION
**Pull request by ealsur**
Adds recommended version and fixes link formatting

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
+
+## Release notes
+
 Preview features are treated as a separate branch and will not be included in the official release until the feature is ready. Each preview release lists all the additional features that are enabled.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-### <a name="3.20.0"/> [3.20.0](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.20.0) - 2021-06-18
+### <a name="3.20.0"/> [3.20.0](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.20.0) - 2021-06-21
```
Comment (j82w) [inline changelog.md:16]: Please add notes from the following PR: https://github.com/Azure/azure-cosmos-dotnet-v3/pull/2502

Added: Diagnostics: Adds Direct TransportRequestStats for tracking transport request timeline
Fixed:  Tracing: Removes noisy trace in Direct mode

Comment (ealsur) [reply changelog.md:16]: Was this missed on the release changelog then?

Comment (ealsur) [reply changelog.md:16]: Added them

Comment (j82w) [reply changelog.md:16]: Yes, it was missed in the release changelog. I was looking through the changelog and noticed it wasn't included.

File: changelog.md [hunk 2]
```diff
@@ -378,7 +392,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - [#1550](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/1550) CosmosOperationCanceledException: This enables users to access the diagnsotics when an operation is canceled via the cancellation token. The new type extends OperationCanceledException so it does not break current exception handling and includes the CosmosDiagnostic in the ToString().
 - [#1578](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/1578) Query: Adds memory optimization to prevent coping the buffer
-- [#1578](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/1578) Query: Adds support for ignore case for [Contains](https://docs.microsoft.com/azure/cosmos-db/sql-query-contains) and [StartsWith](https://docs.microsoft.com/azure/cosmos-db/sql-query-startswith) functions.
+- [#1578](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/1578) Query: Adds support for ignore case for [Contains](/azure/cosmos-db/sql-query-contains) and [StartsWith](/azure/cosmos-db/sql-query-startswith) functions.
```
Comment (j82w) [inline changelog.md:395]: Why did we remove the full url? This will break the github changelog page.

Comment (ealsur) [reply changelog.md:395]: This is a requirement for docs embedded in MSDocs, full URL generates warnings in their builds. 

The main point where customers reach our changelog would be the MSDocs.

Comment (ealsur) [reply changelog.md]: Removed the links

**Review by j82w — APPROVED** [review]

**Pull request merged**